src/strategies/balanced_market.py

```diff
@@ -233,6 +235,12 @@ def step(self, event_list=[]):
 
                 # discharge with maximum power (scaled with power factor)
                 p = -(vehicle.battery.loading_curve.max_power * self.V2G_POWER_FACTOR)
+                # limit to GC discharge power
+                # max unload power is symmetric to max load
+                # -> can unload double max power, minus what is already discharged
+                gc_cur_discharge_power_limit = (timesteps[v2g_ts_idx]["power"]
+                                                - 2*timesteps[v2g_ts_idx]["max_power"])
```

Comment:
Why is it 2*max_power? Shouldnt it just be max_power?
E.G. If `timesteps[v2g_ts_idx]["power"] = 0` no power is available in current time step. In that case the `gc_cur_discharge_power_limit` would be twice as high as the gc charge limit.

Reply:
Didn't appear in my tests, but of course GC limits should be the same positive and negative. @stefansc1